tests/HasRolesTest.php

```diff
@@ -429,6 +445,23 @@ public function it_can_list_all_the_coupled_permissions_both_directly_and_via_ro
         );
     }
 
+    /** @test */
+    public function it_can_scope_users_with_permissions()
```

Comment:
Do you mind moving this test to `HasPermissionsTest`?
Perhaps near https://github.com/spatie/laravel-permission/blob/master/tests/HasPermissionsTest.php#L105